app/helpers/application_helper/toolbar/middleware_messaging_center.rb

```diff
@@ -0,0 +1,19 @@
+# noinspection RubyArgCount
+class ApplicationHelper::Toolbar::MiddlewareMessagingCenter < ApplicationHelper::Toolbar::Basic
+  button_group('middleware_messaging_policy', [
+                 select(
+                   :middleware_messaging_policy_choice,
+                   'fa fa-shield fa-lg',
+                   t = N_('Policy'),
+                   t,
+                   :enabled => "false",
+                   :items   => [
+                     button(
+                       :middleware_messaging_tag,
+                       'pficon pficon-edit fa-lg',
+                       N_('Edit Tags for this Middleware messaging'),
```

Comment:
@jdoyleoss @serenamarie125 I have a general question regarding such strings - should Middleware be with capital letter or not? Messaging - same question - capital or not? (this will of course apply to align in all other entities as well)